core-common/src/main/java/org/glassfish/jersey/logging/LoggingInterceptor.java

```diff
@@ -78,6 +85,7 @@
         add(MediaType.APPLICATION_SVG_XML_TYPE);
         add(MediaType.APPLICATION_XHTML_XML_TYPE);
         add(MediaType.APPLICATION_XML_TYPE);
+        add(APPLICATION_VND_API_JSON);
```

Comment:
I would suggest to move this line (88 - `add(APPLICATION_VND_API_JSON);)` under the line 81 `add(TEXT_MEDIA_TYPE);` to keep that aligned to predefined  and custom constants.  Otherwise looks good to me.

Reply:
Ok, I will do that

Reply:
Fixed, waiting on Travis